wondrous-app/components/Settings/Members/MembersTableRow/styles.tsx

```diff
@@ -0,0 +1,266 @@
+import { MenuItem, Select, Typography } from '@mui/material';
+import styled from 'styled-components';
+import DefaultUserImage from 'components/Common/Image/DefaultUserImage';
+import typography from 'theme/typography';
+import palette from 'theme/palette';
+
+export const UserRowWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 80px;
+`;
+
+export const UserProfile = styled.a`
+  display: flex;
+  align-items: center;
+  gap: 9px;
+  text-decoration: none;
+`;
+
+const ProfilePictureStyles = {
+  width: '40px',
+  height: '40px',
+  borderRadius: '1000px',
+};
+
+export function DefaultProfilePicture(props) {
+  return <DefaultUserImage style={ProfilePictureStyles} />;
+}
+
+export const UserProfileNameContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+`;
+
+export const UserProfileName = styled(Typography)`
```

Comment:
it's the same as `<Typography color={palette.white} fontSize={15} fontWeight={500}>`, you don't need styled-component